### PR TITLE
Changed Documentation. Still could not figure out how to add visualis…

### PR DESCRIPTION
**Pull request by arkadeep16121993**
…ations

### CODE REVIEW

File: DecisionTreeAndRandomForest/Project.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[deps]
+DecisionTreeAndRandomForest = "2e1a1f2f-ab8e-43f0-b75f-c7407e5d21e0"
```
Comment (cedric-len) [inline DecisionTreeAndRandomForest/Project.toml]: I guess this folder and file was created accidentally. Please remove it

File: docs/src/dataset_examples.md [hunk 2]
```diff
@@ -1,111 +1,167 @@
...
-First, ensure to import all the necessary packages: 
+## Setup
 
-```@example 2
-using DataFrames  
-using MLJ: load_iris,unpack,partition
+```
+julia
+using Pkg
+Pkg.activate("DecisionTreeAndRandomForest")
```
Comment (cedric-len) [inline docs/src/dataset_examples.md]: I don't think that this is necessary. It probably even created the additional folder ^^

File: docs/src/dataset_examples.md [hunk 3]
```diff
@@ -1,111 +1,167 @@
...
+Pkg.add(url="https://github.com/marleenlukei/DecisionTreeAndRandomForest.jl/")
+
+```
+
+## Classification and Random Forest - Classifying Iris Types
```
Comment (cedric-len) [inline docs/src/dataset_examples.md]: Should be Classification Tree and Random Forest

File: docs/src/dataset_examples.md [hunk 4]
```diff
@@ -1,111 +1,167 @@
...
+### Decision Tree
+
+Next, we create the decision tree and fit it to the training data:
 
-```@example 2
-tree = DecisionTree(split_gini)
-fit!(tree, train_data, train_labels)  
-nothing # hide
+```julia
+tree = DecisionTree(-1, 1, -1, split_gini)
```
Comment (cedric-len) [inline docs/src/dataset_examples.md:40]: Maybe explain what the hyperparameters are doing

File: docs/src/dataset_examples.md [hunk 5]
```diff
@@ -1,111 +1,167 @@
...
+accuracy = sum(predictions .== test_labels) / length(test_labels)
+println("Decision Tree - Accuracy: ", accuracy)
+```
+
+### Random Forest
+
+Next, we create the random forest and fit it to the training data:
+
+```julia
+forest = RandomForest(-1, 1, split_gini, 10, 0.8, -1)
```
Comment (cedric-len) [inline docs/src/dataset_examples.md]: Same here. Explaining the parameters would be great

File: docs/src/getting_started.md [hunk 6]
```diff
@@ -1,59 +1,159 @@
...
+- **Custom Splitting Criteria**: Support for various splitting criteria such as Gini Impurity, Information Gain, and Variance Reduction.
+
+## Basic Example of a Classification Tree
+
+### Step 1: Setup
+
+```
+julia
+using Pkg
+Pkg.activate("DecisionTreeAndRandomForest")
```
Comment (cedric-len) [inline docs/src/getting_started.md]: Same here. I think this needs to be removed

File: docs/src/getting_started.md [hunk 7]
```diff
@@ -1,59 +1,159 @@
...
-```@example 1
-tree = DecisionTree(split_gini)
-other_tree = DecisionTree(3, 2, split_gini)
-nothing # hide
-```
+Initialize a classification tree:
 
-We can build the tree using the `fit` function.
+```julia
+tree = DecisionTree(-1, 1, -1, split_gini)
```
Comment (cedric-len) [inline docs/src/getting_started.md]: In the getting started guide, i would use as few hyperparameters as possible.

File: docs/src/getting_started.md [hunk 8]
```diff
@@ -1,59 +1,159 @@
...
+data = ["dog" 37.0; "dog" 38.4; "dog" 40.2; "dog" 38.9; "human" 36.2; "human" 37.4; "human" 38.8; "human" 36.2]
+labels = ["healthy", "healthy", "sick", "healthy", "healthy", "sick", "sick", "healthy"]
+```
+
+### Step 2: Initialize a Random Forest
+
+Initialize a random forest with the specified parameters:
+
+```julia
+forest = RandomForest(-1, 1, split_gini, 10, 0.8, -1)
```
Comment (cedric-len) [inline docs/src/getting_started.md]: Same here. For a getting started guide, i think less parameters would be better

File: docs/src/getting_started.md [hunk 9]
```diff
@@ -1,59 +1,159 @@
...
+```
+
+By following these steps, you can create and use a basic random forest. This example illustrates how random forests can be applied to simple datasets for classification tasks.
+
+## Adding More Splitting Criteria
+
+To add more splitting criteria, define a new function that computes the desired criterion. For example, to implement a Chi-Squared Split:
+
+```julia
+using Statistics: chi2
```
Comment (cedric-len) [inline docs/src/getting_started.md]: Remove that import if it isn't used

File: docs/src/dataset_examples.md (hunk 5 above)
Comment (cedric-len) [reply docs/src/dataset_examples.md]: and maybe you can also use a value that is not the default value

File: Project.toml [hunk 10]
```diff
@@ -5,6 +5,8 @@ version = "1.0.0-DEV"
 
 [deps]
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
```
Comment (cedric-len) [inline Project.toml]: Since these dependencies are not used yet i would remove them for now

**Review by cedric-len — CHANGES_REQUESTED** [review]

File: DecisionTreeAndRandomForest/Project.toml (hunk 1 above)
Comment (arkadeep16121993) [reply DecisionTreeAndRandomForest/Project.toml]: Noted. Will delete in my next push

**Review by cedric-len — APPROVED** [review]

**Pull request merged**